src/js/Edit/L.PM.Edit.LayerGroup.js

```diff
@@ -138,4 +139,19 @@ Edit.LayerGroup = L.Class.extend({
   getOptions() {
     return this._options;
   },
+  _getMap(){
+    if(this._map){
+      return this._map
+    }
+
+    for(let i = 0; i < this._layers.length; i+=1){
```

Comment:
Please use `array.find()` instead of this loop. This whole function could be a one-liner I think.



```js
_getMap() {
  return this._map || this._layers.find(l => !!layer._map)?._map || null
}

```